pyramda/function/flip.py

```diff
@@ -0,0 +1,24 @@
+from .curry import (
+    curry,
+    curry_by_spec
+)
+
+from ..private.curry_spec import make_func_curry_spec
+
+
+@curry
+def flip(f):
+    """
+    Calls the function f by flipping the first two positional
+    arguments
+    """
+
+    def flip_first_two(xs):
+        return xs[1::-1] + xs[2::] if len(xs) > 1 else xs
```

Comment:
This can be lifted outside the definition of `flip`, and I think it's more readable that way